content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that identifies one group as deservedly superior to another. Common examples are that men are superior to women (sexism) and one ethnicity is superior to all other ethnicities (racism).  
+
+or  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them special. 
+ 
+or 
+  
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’. 
+For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice. 
+ 
+or
+  
+> Even if *it is the case* we are uniquely superior, morality requires we care for those more vulnerable than ourselves; not that we treat them as a resource.
+
+## Detail - Analytic
+
+> A Definition of Superior (they are many and varied) : of higher rank, quality, or importance - [Merriam-Webster](http://web.archive.org/web/20190711165845/https://www.merriam-webster.com/dictionary/superior)
+
+
+What most people forget (*or choose to overlook*) in using the term superior is that; *We alone* determine the ranking systems. *We alone* determine what denotes better quality. *We alone* determine what makes one thing more important than another.
```

Comment:
Let's expand on why that's an issue. As the (semi) hidden premise is that our arbitrary BS is no better than any other arbitrary BS, stop being so narcissistic and look at the world through other beings eyes for a change. But I think it needs more spelling out than even the HHGTTG quote provides (do you have a bet with someone as to how many HHGTTG quotes you can squeeze in by the way? 😂)

Reply:
I know it's expanded on in the next subsection btw, but it's worth putting a brief nudge in the "why" direction to aid the reading flow.

Reply:
Added a sentence that feeds to the eco-feminism bit to come. Sufficient?

Reply:
Actually.... changing the order a bit.... think it reads better.